src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -496,8 +517,50 @@ public TStructuralType Patch(TStructuralType original)
         /// <summary>
         /// Overwrites the <paramref name="original"/> entity with the values stored in this delta.
         /// </summary>
+        /// <param name="original">The entity to be updated.</param>
+        /// <param name="apiHandler">API Handler for the entity.</param>
+        /// <param name="apiHandlerFactory">API Handler Factory</param>
+        internal void Patch(TStructuralType original, IODataAPIHandler apiHandler, ODataAPIHandlerFactory apiHandlerFactory)
```

Comment:
this method should be public if it's meant to be accessed from the controller action method.

Reply:
Good catch